test-clients/src/main/java/com/hedera/services/bdd/suites/contract/openzeppelin/ERC1155ContractInteractions.java

```diff
@@ -0,0 +1,154 @@
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.hedera.services.bdd.spec.HapiApiSpec.defaultHapiSpec;
+import static com.hedera.services.bdd.spec.queries.QueryVerbs.getAccountBalance;
+import static com.hedera.services.bdd.spec.queries.QueryVerbs.getAccountInfo;
+import static com.hedera.services.bdd.spec.queries.QueryVerbs.getTxnRecord;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.contractCall;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.contractCreate;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.cryptoCreate;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.fileCreate;
+import static com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.updateLargeFile;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.withOpContext;
+
+public class ERC1155ContractInteractions extends HapiApiSuite {
+	private static final Logger log = LogManager.getLogger(ERC1155ContractInteractions.class);
+	private static final String CONTRACT_NAME = "ERC-1155";
+	private static final String OPERATIONS_PAYER = "payer";
+	private static final String ACCOUNT1 = "acc1";
+	private static final String ACCOUNT2 = "acc2";
+	private static final String CONTRACT_FILE_NAME = "contractFileName";
+	public static void main(String... args) {
+		new ERC1155ContractInteractions().runSuiteSync();
+	}
+	
+	private ByteString getFileContents(String path) {
+		ByteString contents = null;
+		try {
+			var bytes = Files.readAllBytes(Path.of(path));
+			contents = ByteString.copyFrom(bytes);
+		} catch (IOException e) {
+			e.printStackTrace();
+		}
+		return contents;
+	}
+	
+	@Override
+	protected List<HapiApiSpec> getSpecsInSuite() {
+		return List.of(
+				erc1155()
+		);
+	}
+
+	private HapiApiSpec erc1155() {
+		final var contents = getFileContents(ContractResources.ERC_1155_BYTECODE_PATH);
+		return defaultHapiSpec(CONTRACT_NAME)
+				.given(
+						cryptoCreate(ACCOUNT1),
+						cryptoCreate(ACCOUNT2),
+						cryptoCreate(OPERATIONS_PAYER).balance(ONE_MILLION_HBARS),
+						fileCreate(CONTRACT_FILE_NAME).payingWith(OPERATIONS_PAYER),
+						updateLargeFile(OPERATIONS_PAYER, CONTRACT_FILE_NAME, contents)
+				)
+				.when()
+				.then(
+						contractCreate(CONTRACT_NAME).bytecode(CONTRACT_FILE_NAME).via("contractCreate").payingWith(OPERATIONS_PAYER),
+						getTxnRecord("contractCreate").logged(), // 121618 gas
+						getAccountBalance(OPERATIONS_PAYER).logged(), // started with 1M hbars
+						getAccountInfo(ACCOUNT1).saveToRegistry(ACCOUNT1 + "Info"),
+						getAccountInfo(ACCOUNT2).saveToRegistry(ACCOUNT2 + "Info"),
+						getAccountInfo(OPERATIONS_PAYER).saveToRegistry(OPERATIONS_PAYER + "Info"),
```

Comment:
saveToRegistry is renamed to savingSnapshot with the latest changes and the build fails